src/types/model.ts

```diff
@@ -122,6 +122,7 @@ export interface Institution {
   updateUser: number;
   updateDate: Date;
   signingOfficials: SimplifiedDuosUser[];
+  domains?: string[];
```

Comment:
Is this really optional?

Reply:
This is true. In general, null values are not returned by the API. This can be changed, but it's the default. There are many institutions that do not have domains yet.

Reply:
I didn't realize that we weren't populating domains for all institutions.  Thanks for taking the time to explain!